src/rust/engine/options/src/env.rs

```diff
@@ -6,10 +6,11 @@ use std::collections::HashMap;
 use std::env;
 use std::ffi::OsString;
 
-use super::id::{NameTransform, OptionId, Scope};
+use super::id::{NameTransform, OptionId};
 use super::{DictEdit, OptionsSource};
 use crate::fromfile::FromfileExpander;
 use crate::parse::Parseable;
+use crate::scope::Scope;
```

Comment:
I guess we don't have a preferred idiom for `use super` vs `user crate`? I prefer the latter as it makes refactoring easier.

Reply:
Not really. "It depends." 😃